--- origsrc/xf86-video-nested/src/Makefile.am	2012-03-26 23:31:23.000000000 -0500
+++ src/xf86-video-nested/src/Makefile.am	2012-03-26 23:35:06.297228300 -0500
@@ -24,7 +24,8 @@
 AM_CFLAGS = $(XORG_CFLAGS) $(PCIACCESS_CFLAGS) $(X11_CFLAGS) $(XEXT_CFLAGS)
 
 nested_drv_la_LTLIBRARIES = nested_drv.la
-nested_drv_la_LDFLAGS = -module -avoid-version $(X11_LIBS) $(XEXT_LIBS)
+nested_drv_la_LDFLAGS = -module -avoid-version
+nested_drv_la_LIBADD = $(X11_LIBS) $(XEXT_LIBS) $(XORG_LIBS)
 nested_drv_ladir = @moduledir@/drivers
 
 nested_drv_la_SOURCES = driver.c nested_input.c nested_input.h xlibclient.c client.h
